### ISSUE TITLE
relayer start panic

### ISSUE THREAD

**Issue by desa** [cosmos/relayer#579]
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x4caaa03]

goroutine 9 [running]:
github.com/cosmos/relayer/relayer.(*RelayMsgs).SendWithController(0xc0012f86e0, 0xc000d137a0, 0xc000f4a1b0, 0xb0)
	/Users/michaeldesa/go/src/github.com/strangelove-ventures/relayer/relayer/relayMsgs.go:168 +0x883
github.com/cosmos/relayer/relayer.(*RelayMsgs).Send(...)
	/Users/michaeldesa/go/src/github.com/strangelove-ventures/relayer/relayer/relayMsgs.go:64
github.com/cosmos/relayer/relayer.RelayPackets(0xc000d137a0, 0xc000f4a1b0, 0xc001260210, 0x200000, 0x5)
	/Users/michaeldesa/go/src/github.com/strangelove-ventures/relayer/relayer/naive-strategy.go:369 +0x2e8
github.com/cosmos/relayer/relayer.StartRelayer.func1()
	/Users/michaeldesa/go/src/github.com/strangelove-ventures/relayer/relayer/strategies.go:29 +0x2ef
created by github.com/cosmos/relayer/relayer.StartRelayer
	/Users/michaeldesa/go/src/github.com/strangelove-ventures/relayer/relayer/strategies.go:11 +0xc5
```

**Comment by jtieri** [comment]
Thanks for opening an issue @desa 

We are in the final stages of pushing out a v2 of the go-relayer, this issue is addressed in the new relayer and the `rly start` command should work as intended now. If you wanted to check this new work out before we have a tagged release candidate you could checkout from the branch in this PR https://github.com/cosmos/relayer/pull/578

**Comment by hoang14x** [comment]
hi @jtieri 
I query balance and have error
` rly q balance  ibc-0 acc_0  --home gaid01`

`panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x1034c6ea8]

goroutine 1 [running]:
github.com/cosmos/relayer/cmd.queryBalanceCmd.func1(0x14000f4ec80, {0x14000f27d40, 0x2, 0x4})
        /Users/lap02139/Documents/Work2/relayer/cmd/query.go:239 +0x28
github.com/spf13/cobra.(*Command).execute(0x14000f4ec80, {0x14000f27d00, 0x4, 0x4})
        /Users/lap02139/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x668
github.com/spf13/cobra.(*Command).ExecuteC(0x14000f11680)
        /Users/lap02139/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/lap02139/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/cosmos/relayer/cmd.Execute()
        /Users/lap02139/Documents/Work2/relayer/cmd/root.go:107 +0x40
main.main()
        /Users/lap02139/Documents/Work2/relayer/main.go:6 +0x20
`

**Comment by jtieri** [comment]
Closing this as I'm pretty sure the issue was fixed in #578
